### ISSUE TITLE
CTRL+Z doesn't edit back

### ISSUE THREAD

**Issue by TCarmine** [microsoft/vscode#74834]
This seems to be a bug since ages.

Context:

Ubuntu 18.04 on Lenovo legion y520-15ikbn.

Version: 1.34.0
Commit: a622c65b2c713c890fcf4fbf07cf34049d5fe758
Date: 2019-05-15T21:55:35.507Z
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.15.0-20-generic snap 

This bug appeared 

- VSCode Version:  1.34.0
- OS Version: Ubuntu 18.04

Steps to Reproduce:

1. Open a file, do some change
2. Try to edit from command line pressing CTRL+Z

Does this issue occur when all extensions are disabled?: Yes/No
Yes. It does still appear once all extensions are disabled.

I indeed in order to test disabled all of them.

**Comment by gjsjohnmurray** [comment]
Does the Undo option from the Edit menu work? If yes, maybe you have bound Ctrl+Z to a different command. Launch "Keyboard Shortcuts" from context menu of gearwheel near bottom left corner. Click "Record Keys" icon. Type Ctrl+Z. What command gets listed?

**Comment by egamma** [comment]
Please also see this guide https://github.com/microsoft/vscode/wiki/Keybinding-Issues for how to track down keybinding issues.

**Comment by TCarmine** [comment]
@gjsjohnmurray  yes it does work from the menu. Also on the Keyboard Shortcuts it is the right "binding".

By the way I found the issue. I do have a German keyboard, and though I set it in the OS( to use the German keyboard mapping) then as CTRL+Y works as CTRL-Z ( in German Z and Y positions are switched) vscode is "see" my keyboard as an International USA one, that make sense as I guess is the default.
